azurerm/resource_arm_mssql_elasticpool.go

```diff
@@ -227,6 +229,7 @@ func resourceArmMsSqlElasticPoolCreateUpdate(d *schema.ResourceData, meta interf
 		Tags:     expandTags(tags),
 		ElasticPoolProperties: &sql.ElasticPoolProperties{
 			PerDatabaseSettings: expandAzureRmMsSqlElasticPoolPerDatabaseSettings(d),
+			ZoneRedundant:       &zoneredundant,
```

Comment:
I think we should be conditionally setting this:

```go
if v, ok := d.GetOkExists("zone_redundant").(bool); ok {
	elasticPool. ElasticPoolProperties.ZoneRedundant = &v
}
```

This is because if someone has set the value via the portal or other means and not updated there terraform when we release this, we'll override their value.

Reply:
thanks for this hint. didn't noticed it.